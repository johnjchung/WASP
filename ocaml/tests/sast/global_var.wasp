int a = 4;

Endpoint e(GET:) {
    int b = a;
}
