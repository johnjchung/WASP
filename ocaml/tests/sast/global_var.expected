int a = [int] 4;Endpoint e( GET:) {int b = [int] a;}
